.eslintrc.json

```diff
@@ -1,9 +1,11 @@
 {
     "extends": "airbnb",
-    "plugins": ["jest"],
     "parser": "babel-eslint",
     "rules": {
       "semi": [2, "never"],
       "arrow-body-style": [0, "as-needed"]
+    },
+    "env": {
+      "jest": true
```

Comment:
Should "env" indeed contain "jest"? Why?

Reply:
Yes, this just adds all the jest related stuff such as functions to the whole environment, so lint knows that test() functions for example do exist and are declared.